app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,74 @@
+class ReviewsController < ApplicationController
```

Comment:
I am assuming this was rails generated. This will need to be fleshed out.

Reply:
Yes.